docs/aws/audit/iamcompliance/rules/account_with_minimum_admins.mdx

```diff
@@ -23,6 +23,266 @@ CBP
+
+3. For each user, check the "Permissions" tab to see what policies are attached to the user. Look for policies that grant administrator privileges, such as "AdministratorAccess".
+
+4. Count the number of users with administrator privileges. If the number is less than the minimum number of admins you have set as a standard for your organization, then it is a misconfiguration.
+</Accordion>
+
+<Accordion title='Using CLI'>
+1. Install and configure AWS CLI: Before you can start using AWS CLI, you need to install it on your local machine and configure it with your AWS account credentials. You can do this by running the following commands:
+
+   Installation:
+   ```
+   pip install awscli
+   ```
+   Configuration:
+   ```
+   aws configure
+   ```
+   You will be prompted to provide your AWS Access Key ID, Secret Access Key, Default region name, and Default output format.
+
+2. List IAM Users: Once AWS CLI is set up, you can list all the IAM users in your AWS account by running the following command:
+   ```
+   aws iam list-users
+   ```
+   This command will return a list of all IAM users along with their details.
+
+3. List IAM Policies: For each user, you can list the policies attached to them. This can be done using the following command:
+   ```
+   aws iam list-user-policies --user-name <username>
+   ```
+   Replace `<username>` with the name of the user. This command will return a list of all policy names attached to the specified user.
+
+4. Check for Admin Policies: Finally, you can check if the user has 'AdministratorAccess' policy attached. This policy provides full access to AWS services and resources. If the number of users with this policy is less than the minimum required, it indicates a misconfiguration. You can do this by checking if 'AdministratorAccess' is present in the list of policies returned in the previous step.
+</Accordion>
+
+<Accordion title='Using Python'>
+1. Install the necessary AWS SDK for Python (Boto3) if you haven't done so already. You can install it using pip:
+
+```python
+pip install boto3
+```
+
+2. Import the necessary modules and create a session using your AWS credentials:
+
+```python
+import boto3
+
+session = boto3.Session(
+    aws_access_key_id='YOUR_ACCESS_KEY',
+    aws_secret_access_key='YOUR_SECRET_KEY',
+    region_name='YOUR_REGION'
+)
+```
+
+3. Use the IAM client from Boto3 to list all the IAM users in your AWS account:
+
+```python
+iam = session.client('iam')
+
+users = iam.list_users()['Users']
+```
+
+4. Iterate over the users and check their policies. If a user has the 'AdministratorAccess' policy, increment a counter. If the counter is less than the minimum number of admins you want, raise an alert:
+
+```python
+admin_count = 0
+for user in users:
+    policies = iam.list_attached_user_policies(UserName=user['UserName'])['AttachedPolicies']
+    for policy in policies:
+        if policy['PolicyName'] == 'AdministratorAccess':
+            admin_count += 1
+
```

Comment:
<h2>Secret Detected: Secret Keyword</h2> <a href="https://console.cloudanix.com/code?t=secrets"> Click here for more details</a>